test/functional/p2p_blockfilters.py

```diff
@@ -47,11 +47,25 @@ def set_test_params(self):
         self.rpc_timeout = 480
         self.num_nodes = 2
         self.extra_args = [
-            ["-blockfilterindex", "-peerblockfilters"],
-            ["-blockfilterindex"],
+            ["-blockfilterindex=basic", "-peerblockfilters"],
+            ["-blockfilterindex=basic", "-peerblockfilters=0"],
         ]
 
     def run_test(self):
+        self.stop_node(0)
+
+        self.log.info('Check that -peerblockfilters without -blockfilterindex raises init error')
+        err_msg = 'Error: Cannot set -peerblockfilters without -blockfilterindex.'
+        self.nodes[0].assert_start_raises_init_error(['-blockfilterindex=0', '-peerblockfilters'], err_msg)
+
+        self.log.info('Check that passing unknown -blockfilterindex type raises init error')
```

Comment:
nit: here, above line 57, and below line 212, I prefer the less-verbose "Test..." over "Check that..." and it's also probably the most frequently used term in the functional test logging. It's subjective though, apart from being less long.